### PR TITLE
Blog Post for PyCon Pune-2017

### PR DESCRIPTION
**Pull request by sbulage**
Written blog post for PyCon Pune-2017.

### CODE REVIEW

File: site/_posts/2017-03-22-pycon-pune-2017.html.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Conclusion of an event :
+
+On that note, we had learned, enjoyed and tried to share as much information as possible about *ManageIQ* with a lot of enthusiastic people during the two days at [PyCon Pune 2017](https://pune.pycon.org/).
+
+Too many people were interested to get more information about *ManageIQ*, and they shared their contact information with us. In future, we will be planning meetups for them.
+
+## Wrapping up :
+
+This was a quick overview of what have we done in PyCon Pune-2017, but feel free to check out our event pics on [twitter feed](https://twitter.com/manageiq).
```
Comment (cybette) [inline site/_posts/2017-03-22-pycon-pune-2017.html.md]: Might be better to link to the twitter moment for pics, i.e. change 

`on [twitter feed](https://twitter.com/manageiq)`

to

`in this [twitter moment](https://twitter.com/i/moments/834117124872364033)`

**Comment by cybette** [comment]
Great job @sbulage, nice post!

@JPrause can you take a look-see as well?

File: site/_posts/2017-03-22-pycon-pune-2017.html.md (hunk 1 above)
Comment (sbulage) [inline site/_posts/2017-03-22-pycon-pune-2017.html.md]: @cybette sure I will update it.

**Comment by JPrause** [comment]
Nice,...LGTM! 👍

**Comment by sbulage** [comment]
@cybette any other correction/changes ...?

**Comment by cybette** [comment]
@sbulage no more, looks great! merging...

**Pull request merged**